personal_graph/clients.py

```diff
@@ -35,16 +35,18 @@ class LiteLLMEmbeddingClient(APIClient):
     dimensions: int = 384
```

Comment:
`model_name: str = "openai/text-embedding-3-small"` and dimensions dont seem to be required or used